dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipUdpConnLink.java

```diff
@@ -0,0 +1,492 @@
+/*******************************************************************************
+ * Copyright (c) 2008, 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.sip.stack.transport.sip.netty;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.*;
+
+import com.ibm.sip.util.log.Log;
+import com.ibm.sip.util.log.LogMgr;
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.ffdc.FFDCFilter;
+import com.ibm.ws.sip.container.pmi.PerformanceMgr;
+import com.ibm.ws.sip.parser.util.InetAddressCache;
+import com.ibm.ws.sip.properties.SipPropertiesMap;
+import com.ibm.ws.sip.properties.StackProperties;
+import com.ibm.ws.sip.stack.context.MessageContext;
+import com.ibm.ws.sip.stack.dispatch.Dispatcher;
+import com.ibm.ws.sip.stack.transaction.transport.UseCompactHeaders;
+import com.ibm.ws.sip.stack.transaction.transport.connections.SipMessageByteBuffer;
+import com.ibm.ws.sip.stack.transaction.util.ApplicationProperties;
+import com.ibm.ws.sip.stack.util.StackTaskDurationMeasurer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.socket.DatagramPacket;
+import jain.protocol.ip.sip.ListeningPoint;
+
+//TODO Liberty import com.ibm.ws.management.AdminHelper;
+
```

Comment:
Something we need to address? Along with the todo below or are these from the original class?

Reply:
2do above deleted; 2do below is carried over from chfw